File: NCPC/2017/ncpc2017-packages/judgingmoose/submissions/accepted/per.rb
# I have no idea what I'm doing.

nums = STDIN.read.split().map(&:to_i)
l = nums[0]
r = nums[1]
val = 2 * nums.max
if val == 0
  puts "Not a moose"
elsif l == r
  puts "Even " + val.to_s
else
  puts "Odd " + val.to_s
end

File: NCPC/2017/ncpc2017-packages/judgingmoose/submissions/accepted/per.js
// I have no idea what I'm doing.

var nums = readline().split(' ');
var l = parseInt(nums[0]);
var r = parseInt(nums[1]);
var val = 2 * Math.max(l, r);
if (val == 0) {
    print("Not a moose");
} else if (l == r) {
    print("Even " + val);
} else {
    print("Odd " + val);
}
